addons/lifecycle/functions/fnc_initConfig.sqf

```diff
@@ -125,3 +125,14 @@ private _settingsGroup = ["GRAD Civilians", "2) lifecycle - performance & spawni
     },
     false
 ] call CBA_fnc_addSetting;
+
+[
+    QGVAR(clockrate),
```

Comment:
semantic nitpick: a *rate* is a count over time (like, X per second) whereas this setting here is a time value (delay or timeout in seconds).

Either:
* we rename this var accordingly (maybe "spawnDelay")?, 

Or:

* change description to "spawn attempts per second" and 
* a value of `1 / GVAR(clockrate)` in the CBA_fnc_addPerFrameHandler call below

Reply:
I'll just go ahead and do the latter one, I like that